content/outcomes/cloud-native-development/general__tdd.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Test-Driven Development
```

Comment:
Again, we need to resolve having two articles for the same/similar content when #988 is merged.

Reply:
Reference added!

The content here is way more directed towards practical knowledge and concrete practices with Spring / Cucumber ...